NATIVE_SDK_VERSIONS.md

```diff
@@ -0,0 +1,5 @@
+| Kotlin Multiplatform | iOS SDK | Android SDK |
```

Comment:
seems it doesn't cover `0.4.0`, `0.5.0` and `1.0.0` releases?

Reply:
You are right, my bad. 

I copied the table over from Confluence, looks like it was also outdated there: https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/2271380441/Cross+Platform+SDKs+Versions#Kotlin-Multiplatform

I'll update it right away, thanks for raising 🙇

Reply:
Changes at [88ecbf2](https://github.com/DataDog/dd-sdk-kotlin-multiplatform/pull/164/commits/88ecbf247b4f64e4b7632bfe8921b99fe885f3d0)